packages/flame/lib/src/sprite.dart

```diff
@@ -54,11 +62,31 @@ class Sprite {
     src = (position ?? Vector2.zero()).toPositionedRect(srcSize);
   }
 
+  /// Same as [render], but takes both the position and the size as a single
+  /// [Rect].
+  ///
+  /// **Note**: only use this if you are already using [Rect]'s to represent both the position and
+  /// dimension of your [Sprite]. If you are using [Vector2]s, prefer the other method.
+  void renderRect(
+    Canvas canvas,
+    Rect rect, {
+    Paint? overridePaint,
+  }) {
+    render(
+      canvas,
+      position: rect.topLeft.toVector2(),
+      size: rect.size.toVector2(),
+      overridePaint: overridePaint,
+    );
+  }
+
   /// Renders this sprite onto the canvas.
   ///
   /// * position: x,y coordinates where it will be drawn; default to origin.
-  /// * size: width/height dimensions; it can be bigger or smaller than the original size -- but it defaults to the original texture size.
-  /// * overridePaint: paint to use. You can also change the paint on your Sprite instance. Default is white.
+  /// * size: width/height dimensions; it can be bigger or smaller than the original size -- but it
```

Comment:
I know this isn't really part of your PR, but we should annotate these with dartdocs

Reply:
ah good catch! missed that